dojo/api_v2/serializers.py

```diff
@@ -1002,6 +1002,8 @@ def save(self, push_to_jira=False):
                                            data['scan_type'],)
         except ValueError:
             raise Exception('FileParser ValueError')
+        except:
+            raise Exception('Error while parsing the report, did you specify the correct scan type ?')
```

Comment:
I think this hides the original exception? should it be `raise Exception('blabla', e)` or something?

Reply:
Hmm just read that it was on purpose to catch it in the viewset. This was on our list for a long time to make error handling better, so thanks for doing this.